eng/pipelines/coreclr/pr.yml

```diff
@@ -42,6 +42,20 @@ jobs:
     jobParameters:
       testGroup: innerloop
 
+#
+# Debug library builds
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
+    buildConfig: Debug
```

Comment:
Does it really matter having Debug vs Release builds of the libraries product? The API surface area doesn't change at all. 

I don't fully know what it is used on CoreCLR for, but it might not be needed to have Debug builds as well.

Reply:
I think we should push to always use a release build of libraries in coreclr, even for a debug build of coreclr. If plumbing this through is more work at the moment then we can work on this after the change goes in.

Reply:
I don't think it shouldn't be much work. It is just a matter of including this platforms on the Release template expansion and passing down the correct parameter for liveCoreClrConfiguration set to Release.

Reply:
Actually there's already a Release builds section for libraries which is building these two platforms, so we should just update all coreclr build that depend on libraries to use Release bits.

Reply:
Thanks for pointing that out, fixed.